msi/GitProduct.wxs

```diff
@@ -58,11 +62,25 @@
     <SetProperty Id="ENABLEFSCACHE" Value="[CMDLINE_ENABLEFSCACHE]" After="AppSearch" Sequence="first">CMDLINE_ENABLEFSCACHE</SetProperty>
 
     <!-- ENABLEGCM Property implementing the "Remember Property" pattern -->
-    <Property Id="ENABLEGCM" Value="1">
+    <!-- Initially set this to disabled until we can determine that .NET Framework 4.5.1 and above is installed. -->
+    <Property Id="ENABLEGCM" Value="0" />
```

Comment:
Hehe, I had to think about this a bit. Yeah, this looks like it will work, too, and may in fact be a bit simpler. Thanks. I'll try it out and see how it works when "translated" to WiX.